### ISSUE TITLE
Add calibration curve (reliability diagram) to evaluation

### ISSUE THREAD

**Issue by ashishshiwlani** [ashishshiwlani/hospital-readmission-risk#2]
Clinical risk models must be well-calibrated. Add sklearn.calibration.calibration_curve plot and Expected Calibration Error (ECE) metric to the training report.
